.github/workflows/cpp-python-build.yml

```
@@ -89,8 +92,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          name: ${{ github.job}} rolling release
-          tag_name: ${{ github.job }}
+          name: ${{ github.job}} rolling release for ${{ matrix.os }}
+          tag_name: ${{ github.job }}-${{ matrix.os }}
```

```suggestion
          name: ${{ github.job}} rolling release
          tag_name: ${{ github.job }}
```
I would prefer to release everything under the `Linux rolling release` rather than have a different release for each Ubuntu version: since the package file names are tagged with the Ubuntu version, there is no possible confusion between 18.04 packages and 20.04 packages.